src/web/src/components/Posts/Post.tsx

```diff
@@ -286,6 +286,39 @@ const extractBlogClassName = (url: string) => {
 // a 'guid' from a YouTube video is usually written as 'yt:video:id'
 const extractVideoId = (post: Post): string => post.guid.split(':')[2];
 
+function handleZoom(e: MouseEvent) {
+  let canZoom = 1;
+  // Zoom out of all the currently zoomed images on click
+  document.querySelectorAll<HTMLElement>('.zoomed-image').forEach((zoomedImg) => {
+    zoomedImg.classList.remove('zoomed-image');
+    // if the clicked element is a zoomed image, do not allow further zooming
+    if (zoomedImg === e.target) canZoom = 0;
+  });
+  // Check if the clicked element is an image or an anchor that contains an image
+  let image = null;
+  if (e.target instanceof HTMLImageElement) {
+    image = e.target;
+  } else if (
```

Comment:
On a click event, the event handler of a child is called first before the event is bubbled up to its parent. So isn't this case already handled in the previous statement?